### PR TITLE
Create search API test coverage

### PR DESCRIPTION
**Pull request by nshumoogum**
### What

Test coverage for search API

### How to review

Run tests, making sure they pass, check test covers various scenarios, check dp-search-api spec if need be.

### Who can review

Team B

### CODE REVIEW

File: searchAPI/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+elasticsearch
+dataset API
+search API
+```
+
+Both API's will need to have the following configuration:
+
+```
+export MONGODB_DATABASE=test
+export MONGODB_FILTERS_DATABASE=test
```
Comment (Matt-Guest) [inline searchAPI/README.md]: Is `MONGODB_FILTERS_DATABASE` needed?

File: searchAPI/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+#### Services and software
+
+The following software needs to be running for acceptance tests to be able to
+pass:
+
+```text
+mongo db
+zookeeper
+kafka
+elasticsearch
```
Comment (Matt-Guest) [inline searchAPI/README.md]: what elastic version do you need?

File: searchAPI/get_dimension_search_test.go [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+				response.Value("limit").Equal(2)
+				response.Value("offset").Equal(2)
+			})
+		})
+	})
+
+	// delete mongo test data
+	if err := mongo.Teardown(datasetDoc, editionDoc, versionDoc); err != nil {
+		if err != mgo.ErrNotFound {
+			os.Exit(1)
```
Comment (Matt-Guest) [inline searchAPI/get_dimension_search_test.go]: Should the error be logged before the tests close

**Review by Matt-Guest — COMMENTED** [review]
I wasn't able to run the tests on the search-api. What branches do i need to checkout?

File: searchAPI/README.md (hunk 2 above)
Comment (nshumoogum) [reply searchAPI/README.md]: elasticsearch version 5 and above

**Review by Matt-Guest — APPROVED** [review]

**Pull request merged**